ch04.adoc

```diff
@@ -321,7 +321,7 @@ It is recommended that a reference datetime before the discontinuity should not
 __The **`standard`** calendar is used both for (mostly observational) data with real-world datetimes, including leap seconds (after 1972), and for data not including leap seconds__, generated by models according to the Gregorian calendar.
 Therefore, __when using data written with the **`standard`** calendar, you should regard datetimes and time coordinates as uncertain by a number of seconds (but less than a minute),__ because leap seconds may have been counted when you expected them to have been ignored, or ignored when you expected them to have been counted.
 For existing datasets, this uncertainty can be eliminated only if you have separate information about how leap seconds were treated.
-For producing new datasets, where precision to the second is important, it is recommended to use the **`utc`** calendar for data that follows the real-world calendar including leap seconds, and the **`proleptic_gregorian`** calendar for model-generated data that follows the Gregorian calendar without leap seconds.
+For producing new datasets, where precision to the second is important, it is recommended to use the **`utc`** calendar for data that follows the real-world calendar according to UTC including leap seconds, and the **`proleptic_gregorian`** calendar for model-generated data that follows the Gregorian calendar without leap seconds.
```

Comment:
maybe a couple commas?

For producing new datasets, where precision to the second is important, it is recommended to use the **`utc`** calendar for data that follows the real-world calendar, according to UTC, including leap seconds, and the **`proleptic_gregorian`** calendar for model-generated data that follows the Gregorian calendar without leap seconds.